File: SIMPLE PROGRAMS/ToFindOutBiggestNumberOutOfThreeNumbers/ToFindOutBiggestNumberOutOfThreeNumbers.cpp
#include<iostream>

class ToFindOutBiggestNumberOutOfThreeNumbers
{
public:
    int bigger(int a,int b,int c);
};
int ToFindOutBiggestNumberOutOfThreeNumbers::bigger(int a,int b,int c){
        if((a>b)&&(a>c)){return a;}

        else if((b>a)&&(b>c)){return b;}

        else{return c;}
}

int main(){

     ToFindOutBiggestNumberOutOfThreeNumbers key;

    int a[4],p;

    for(int i=1;i<=3;i++){
        printf("ENTER THE %d VALUE ",i);
        scanf("%d",(a+i));
    }
    p = key.bigger(*(a+1),*(a+2),*(a+3));

    std::cout<<"THE BIGGEST VALUE IS : "<<p;

return 0;
}

File: SIMPLE PROGRAMS/ToFindOutBiggestNumberOutOfThreeNumbers/ToFindOutBiggestNumberOutOfThreeNumbers.java
import java.util.Scanner;

class ToFindOutBiggestNumberOutOfThreeNumbers{

    Scanner io = new Scanner(System.in);

    int a,b,c;
    
    ToFindOutBiggestNumberOutOfThreeNumbers(){
        System.out.println("THIS IS A BIGGEST VALUE IN GIVEN : "+check());

    }
    int check(){

        System.out.print("ENTER THE FIRST VALUE    :");
        this.a=io.nextInt();
        System.out.print("ENTER THE SECOND VALUE   :");
        this.b=io.nextInt();
        System.out.print("ENTER THE THIRD VALUE    :");
        this.c=io.nextInt();

        if((a>b)&&(a>c)){
            return a;
        }
        else if((b>=a)&&(b>=c)){
            return b;
        }
        else{
            return c;
        }
    }
    
    
    public static void main(String []argu){
        ToFindOutBiggestNumberOutOfThreeNumbers key = new ToFindOutBiggestNumberOutOfThreeNumbers();
    }

}